src/Framework/Config/GacelaFileConfig/Factory/GacelaConfigFromBootstrapFactory.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Gacela\Framework\Config\GacelaFileConfig\Factory;
+
+use Gacela\Framework\Config\GacelaConfigBuilder\ConfigBuilder;
+use Gacela\Framework\Config\GacelaConfigBuilder\MappingInterfacesBuilder;
+use Gacela\Framework\Config\GacelaConfigBuilder\SuffixTypesBuilder;
+use Gacela\Framework\Config\GacelaConfigFileFactoryInterface;
+use Gacela\Framework\Config\GacelaFileConfig\GacelaConfigFile;
+
+final class GacelaConfigFromBootstrapFactory implements GacelaConfigFileFactoryInterface
+{
+    /** @var array<string,mixed> */
+    private array $globalServices;
```

Comment:
Why not the PhpDoc here is?

```php
/**
 * @var array{
 *     config?: callable,
 *     mapping-interfaces?: callable,
 *     suffix-types?: callable,
 * } $configFromGlobalServices
 */
```

Reply:
Because global services are a more abstract concept. This here is a way to let psalm/phpstan know the concrete types for these special keys that I am looking for. It's like a (soft)casting. It wouldn't be correct place it in this line 16.